### PR TITLE
Feetch servo

### PR DESCRIPTION
**Pull request by Fabien-B**
This is a driver for the STS3032 servomotor.
It is not complete yet, but it has the main functionalities.

### CODE REVIEW

File: conf/modules/actuators_sts3032.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <dl_setting var="sts3032_current_id" min="1" max="253" step="1" shortname="ID"/>
+        <dl_setting var="sts3032_future_id" min="1" max="252" step="1" shortname="set id" handler="set_id" header="actuators/actuators_sts3032"></dl_setting>
+        <dl_setting var="sts3032_lock_eprom" min="0" max="1" step="1" shortname="lock EPROM" handler="lock_eprom" header="actuators/actuators_sts3032"></dl_setting>
+        <dl_setting var="sts3032_move" min="0" max="4094" step="1" shortname="move" handler="move" header="actuators/actuators_sts3032"></dl_setting>
+        <dl_setting var="sts3032_response_level" min="0" max="1" step="1" shortname="set response level" handler="set_response_level" header="actuators/actuators_sts3032"></dl_setting>
+      </dl_settings>
+    </dl_settings>
+  </settings>
+  <dep>
+    <depends>uart</depends>
```
Comment (gautierhattenberger) [inline conf/modules/actuators_sts3032.xml:30]: Add `actuators` to dependencies ?
Add `<provides>actuators</depends>` ?

File: sw/airborne/modules/actuators/actuators_sts3032.c [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+void sts3032_read_pos(struct sts3032 *sts, uint8_t id);
+void sts3032_enable_torque(struct sts3032 *sts, uint8_t id, uint8_t enable);
+void sts3032_lock_eeprom(struct sts3032 *sts, uint8_t id, uint8_t lock);
+void sts3032_set_response_level(struct sts3032 *sts, uint8_t id, uint8_t level);
+void sts3032_set_id(struct sts3032 *sts, uint8_t id, uint8_t new_id);
+
+
+void actuators_sts3032_init(void)
+{
+  sts.periph = &(STS3032_DEV);
```
Comment (gautierhattenberger) [inline sw/airborne/modules/actuators/actuators_sts3032.c:90]: apply indentation script

**Review by gautierhattenberger — CHANGES_REQUESTED** [review]
Looks fine. Only minor changes.

**Comment by Fabien-B** [comment]
changes done

**Review by gautierhattenberger — APPROVED** [review]

**Pull request merged**